### ISSUE TITLE
Allow specifying permissions when pushing to chart repository

### ISSUE THREAD

**Issue by elevate-andrewlock** [hypnoglow/helm-s3#37]
In our current setup we have two Amazon accounts, AccountA and AccountB. Charts are pushed using the s3 plugin to a bucket repo by AccountA, but AccountB needs to install charts from the repo. Unfortunately, as AccountA owns the objects, this isn't possible. 

We effectively have [this scenario](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example4.html). The only way around it is to use the AccountA to explicitly grant permission to AccountB on the files after pushing them, or by using an acl parameter, e.g.

```
aws s3api put-object --bucket examplebucket --key chart.tgz --grant-full-control id="canonicalUserId-ofAccountA" --body ...
```

when pushing the chart up initially. 

What do you think about the ability to customise the permissions when pushing charts (and the index file)?

**Comment by mvisonneau** [comment]
facing the same issue at the moment :(

**Comment by hypnoglow** [comment]
This should be resolved with #43
